requirements.txt

```diff
@@ -2,4 +2,8 @@ pytest
 pandas
 validate_email
 requests
-teradatasql
\ No newline at end of file
+teradatasql
```

Comment:
@dmedora could you pin all dependencies by using `pip freeze`? We should pin all or pin none. Since it's a standalone application I suggest pinning all dependencies.

Reply:
Pinned versions for all.